### DISCUSSION TITLE
create3 webapi?

### DISCUSSION THREAD

**Discussion by framlin** [iRobotEducation/create3_docs#284] [Questions]
Hello,
I just saw the serial-communication-tutorial and there I learned, that the create3-webserver offers a serial-config - service - url.
Are there more such services? Is there a documentation of the create3 webapi?

**Comment by kwujciak** [comment]
[Here](https://iroboteducation.github.io/create3_docs/webserver/serial-config/) is some documentation on the webserver which talks a bit about serial-config and other services.
